Windows/ppsspp.rc

```diff
@@ -445,6 +445,13 @@ BEGIN
         MENUITEM "Memory View...",                          ID_DEBUG_MEMORYVIEW
     END
 
+    POPUP "Movie"
+    BEGIN
+        MENUITEM "Dump Frames",                             ID_MOVIE_DUMPFRAMES
+        MENUITEM "Lossless Codec (FFV1)",                   ID_MOVIE_USEFFV1
+		MENUITEM "Dump Audio",                              ID_MOVIE_DUMPAUDIO

```

Comment:
In this file we use spaces.  Sorry, this isn't in the editorconfig yet, should add...

-[Unknown]